### ISSUE TITLE
corexy

### ISSUE THREAD

**Issue by DoruKosztner** [ultimachine/Einsy-Rambo#8]
Is this board able to control a corexy printer? Thank you.

**Comment by sjkelly** [comment]
Yes. It runs Marlin, so you would need to just enable the CoreXY kinematics in Configuration.h:
https://github.com/MarlinFirmware/Marlin/blob/00075990d062088f1a291f0546295c1d36821c63/Marlin/Configuration.h#L849
